docs/src/pages/[platform]/components/rating/RatingPropControls.tsx

```diff
@@ -37,15 +37,17 @@ export const RatingPropControls: RatingPropControlsInterface = ({
       <TextField
         label="value"
         value={value}
+        type="number"
         placeholder="Set Value"
-        onChange={(event) => setValue(parseInt(event.target.value))}
+        onChange={(event) => setValue(event.target.value)}
```

Comment:
this doesn't cause a type error?

Reply:
That's what I though, though for some reason it doesn't throw an error. 
<img width="874" alt="Screenshot 2023-06-13 at 4 10 38 PM" src="https://github.com/aws-amplify/amplify-ui/assets/376920/4b11898e-d6b4-4e95-8c8d-ae2b2c157b0c">
I don't think we'd want parseInt though either because it restricts using decimal values like 2.5

Reply:
@calebpollman @reesscot updated (and updated preview video in description)

Reply:
> Looking at the screenshot above, it doesn't look `event` is typed at all, which may be why there was no type error (implicit `any`)

Oh, you're right, thanks. Updating the type for event would have shown the type error then. I added types for event for the controls here that had implicit any type.